### PR TITLE
Updated NPC Eomie quest text

### PR DESCRIPTION
**Pull request by Shad0wCrux**
Updated NPC Eomie quest text - Removed all getitemlinks

### CODE REVIEW

File: world/map/npc/001-1/eomie.txt [hunk 1]
```diff
@@ -64,14 +64,14 @@ L_More:
 
 L_Hurnscald: // player asked how to get Grass Seeds
     mes "[Eomie]";
-    mes "\"Squirrels tend to pick up " + getitemlink("GrassSeed") + " as they gather food. You can find them around Hurnscald if you ride the ferry over.\"";
+    mes "\"Squirrels tend to pick up " + "grass seeds" + " as they gather food. You can find them around Hurnscald if you ride the ferry over.\"";
```
Comment (wushin) [inline world/map/npc/001-1/eomie.txt]: you can remove the " + " out of these lines

File: world/map/npc/001-1/eomie.txt [hunk 2]
```diff
@@ -64,14 +64,14 @@ L_More:
...
     goto L_Close;
 
 L_Mine: // player asked how to get Ice Cubes
     mes "[Eomie]";
     mes "\"Quite the quandary – isn't it? Naturally, you can easily find them on the continent of Kaizei, which is far up north and covered with snow most time of the year. But it might be a bit difficult to get there.\"";
     next;
-    mes "\"I heard that Ice Goblins were seen in the caves surrounding Nivalis. Maybe you can find " + getitemlink("IceCube") + " there.\"";
+    mes "\"I heard that Ice Goblins were seen in the caves surrounding Nivalis. Maybe you can find " + "ice cubes" + " there.\"";
```
Comment (wushin) [inline world/map/npc/001-1/eomie.txt]: same here

File: world/map/npc/001-1/eomie.txt [hunk 3]
```diff
@@ -35,7 +35,7 @@ L_GetStuff:
     // QL_ANWAR == 8
     mes "[Eomie]";
     mes "\"I could try to briefly interrupt my magic and then cast the spell on the fertilizer, but I would need to prepare the grass beforehand.";
-    mes "Bring me " + @grass_amount + " " + getitemlink("GrassSeed") + ", " + @water_amount + " " + getitemlink("BottleOfWater") + ", and " + @ice_amount + " " + getitemlink("IceCube") + ".\"";
+    mes "Bring me " + @grass_amount + " " + "grass seeds" + ", " + @water_amount + " " + "bottles of water" + ", and " + @ice_amount + " " + "ice cubes" + ".\"";
```
Comment (wushin) [inline world/map/npc/001-1/eomie.txt]: you can remove some of the " + " here as well.
mes "Bring me " + @grass_amount + " grass seeds, " + @water_amount + " bottles of water, and " + @ice_amount + " ice cubes.\"";

File: world/map/npc/001-1/eomie.txt (hunk 1 above)
Comment (mekolat) [inline world/map/npc/001-1/eomie.txt]: why is the getitemlink removed? o_o

**Comment by mekolat** [comment]
why remove `getitemlink`? if you can't see it in your client you need to update it, that's all

**Comment by Shad0wCrux** [comment]
While the text and getitemlink doesn't appear in my client it turns out that Fedora 23's ManaPlus version is extremely outdated (1.3.10.27.2 vs current  1.5.11.21), and as I'm told the text appears properly in more recent client versions.

**Pull request closed without merging**